### DISCUSSION TITLE
asyncData in nuxt 3? server-side rendering with Option API in nuxt 3?

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#17300] [General]
The beta has been released but anyone who used `nuxt 2` to do `server-side rendering` was left out, `nuxt 3` currently doesn't support any way to make `server-side` requests without using the `Composition API`?

I have a project in my company waiting and I would love to use `nuxt 3`, but without `asyncData` we are forced to use `nuxt 2`...

Some estimate of when it will be possible to use `asyncData` with `Option API`?

**Comment by simllll** [comment]
Just stumpled upon the same issue, is this intended or acutally even a bug? vue 3 supports option api, therefore I would assume nuxt3 should too?
